Makefile

```diff
@@ -30,7 +30,9 @@ lint:
 
 test:
 	dune runtest --profile=$(PROFILE) --error-reporting=twice -j $(JOBS)
+ifeq ($(PY_TEST), YES)
 	dune build @runtest-python --profile=$(PROFILE)
```

Comment:
I run the python tests when developing, this change will make them more difficult to run. Please change the conditional to make the tests be run by default and when a variable is set they are not run

Reply:
is it not default to run?
PY_TEST is not set by default.

Reply:
when you comes to xs9, you may not run it by default (as xs9 do not have python2 for python test).
I will update to run it by default.